docs/content/docs/build/releases-and-apitags.md

```diff
@@ -37,20 +37,20 @@ There are no guarantees for API breakages or Fluid document compatibility among
 
 ## API Tags
 
-There are no API stability guarantees for packages in the `@fluid-experimental` and `@fluid-internal` scopes.
-`@fluid-experimental` APIs are for developers to try experimental features and provide feedback. It's possible that these APIs can get completely scrapped or drastically changed in a minor release based on developer feedback.
-`@fluid-internal` APIs are only meant for internal consumption by the framework and should not be used.
-
 For packages that are part of the `@fluidframework` scope and the `fluid-framework` package, we use import paths to communicate the stability and guarantees associated with those APIs.
 
--   **Public APIs** - These APIs are officially supported and any breaking changes to these APIs will be done in a major release and go through a deprecation phase before being removed. We guarantee to not break them in minor and patch releases (consistent with the semver guarantees)
```

Comment:
Suggest removing "We guarantee....".  Breaks may happen unintentionally, or we may want to revise our policy in the future.